plugin/storage/es/spanstore/reader.go

```diff
@@ -136,7 +137,12 @@ func NewSpanReader(p SpanReaderParams) *SpanReader {
 
 type timeRangeIndexFn func(indexName string, startTime time.Time, endTime time.Time) []string
 
-type sourceFn func(query elastic.Query, nextTime uint64) *elastic.SearchSource
+type searchAfterFields struct {
+	timestamp uint64
+	spanID    string
```

Comment:
why string and not `model.SpanID`

Reply:
The sort accepts interface arguments. Passing the `modelSpanID` didn't work. It probably wasn't correctly transformed to a string.